src/components/ServiceRequestForm.jsx

```diff
@@ -286,12 +286,18 @@ const ServiceRequestForm = (props, errors, touched) => {
   };
 
   const handleServiceRequestChange = (changeEvent) => {
-    const value = changeEvent.currentTarget.value.toLowerCase();
+    var index = changeEvent.nativeEvent.target.selectedIndex;
```

Comment:
why did this change?

Reply:
I see this nativeEvent repeated over and over again, something seems up.

Reply:
On our previous code base value was actually label string not "ID" . Because ID had special string for Water and Sewer Issues and I believe other . But our select component uses id as a value so the value I am getting from select component is not working for me . I needed as selected label .

Reply:
I think this approach lessens the impact.

```jsx
const { name, options, value, selectedIndex } = changeEvent.target;
const selectedText = options[selectedIndex];
```

Reply:
done